docs/zkapps/front-end-integration-guides/angular.mdx

```diff
@@ -0,0 +1,424 @@
+---
+title: Angular and Github Pages
+description: Getting started with Angular
+keywords:
+  - angular
+  - ui
+  - sharedarraybuffer
+---
+
+# Angular Integration Guide
+## Install a Wallet
+
+- Install a wallet that supports zkApp transactions. For this tutorial, we’ll use **Auro Wallet**. [Download it here](https://www.aurowallet.com/).
+- Add the Auro Wallet browser extension.
+- Open the extension and follow the steps to create a new wallet.
+- Click **"Mainnet"** at the top of the extension view, then select **Show Testnet** from the menu. After that, select **Devnet**.
+    - Using Devnet will allow us to interact with a test version of the Mina network without needing to spend real Mina to pay for transaction fees.
+- Fund your wallet using the [Mina Faucet](https://faucet.minaprotocol.com/).
+    - You'll need to wait about 3 minutes to see the change in balance reflected on chain. You can use [Minascan](https://minascan.io/devnet) to track the status of your transaction.
```

Comment:
```suggestion
    - You'll need to wait one block (~3 minutes) to see the change in balance reflected on chain. You can use [Minascan](https://minascan.io/devnet) to track the status of your transaction.
```
It would be better elaborate the reason for the wait.